canonical-web-and-design/jaas-dashboard
# Add external config

## Done

Converted the config values that were defined in the `.env` file into a `config.js` file so that it can be generated or modified by other hosts, specifically Juju core.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The app should work as normal

## Details

Fixes #353
